source/game_sa/Hud.cpp

```diff
@@ -311,8 +334,8 @@ void CHud::SetHelpMessageStatUpdate(eStatUpdateState state, uint16 statId, float
     m_nHelpMessageStatId = statId;
     m_fHelpMessageStatUpdateValue = diff;
     m_nHelpMessageMaxStatValue = (uint32)max;
-    // *(_WORD *)gString = 2 * (state == 0) + '+';
-    // AsciiToGxtChar(gString, m_pHelpMessage);
+    *(short*)gString = state == STAT_UPDATE_INCREASE ? '+' : '-';
```

Comment:
That's a compiler optimization I think.
In reality it's most likely:
```cpp
gString[0] = state == STAT_UPDATE_INCREASE ? '+' : '-';
gString[1] = 0; // Null terminate
```

Reply:
why not make them strings?
```suggestion
    gString = (state == STAT_UPDATE_INCREASE) ? "+" : "-";
```

Reply:
I belive `gString` is not a pointer, but a c char array.
Perhaps:
```suggestion
    strcpy(gString, (state == STAT_UPDATE_INCREASE) ? "+" : "-");
```